### PR TITLE
fixed runtimewarning in sqrt in vis_metrics.sequential_diff

### PR DESCRIPTION
**Pull request by nkern**
addresses #201

### CODE REVIEW

File: hera_qm/vis_metrics.py [hunk 1]
```diff
@@ -54,13 +54,10 @@ def check_noise_variance(data):
     return Cij
 
 
-def sequential_diff(data, t_int=None, axis=(0,), pad=True, history=''):
+def sequential_diff(data, t_int=None, axis=(0,), pad=True, run_check=True, history=''):
     """
     Take a sequential (forward) difference of a visibility waterfall
```
Comment (plaplant) [inline hera_qm/vis_metrics.py]: Can you make this "one-line summary" actually fit on a single line? When the short-form of the docstring is requested, only the first line is returned, so it's helpful for the summary to fit on a single line.

**Comment by nkern** [comment]
I think the negative values were coming from the fact that `true_divide` returns an `empty` array when `where` is False, and the `empty` array had some stochasticness to it, and it sometimes went to like `10^-256` and `-10^-256`, causing the error.

File: hera_qm/vis_metrics.py [hunk 2]
```diff
@@ -138,7 +137,9 @@ def sequential_diff(data, t_int=None, axis=(0,), pad=True, history=''):
             inv_t_int = np.true_divide(1., inv_sum, where=where)
 
             # get noise correction factor
-            corr = np.sqrt(np.true_divide(inv_t_int, t_int, where=where))
+            corr = np.true_divide(inv_t_int, t_int, where=where)
+            corr[np.isclose(corr, 0.0) + (corr < 0) + np.isnan(corr)] = 0.0   # fixes invalid value in sqrt RuntimeWarning
```
Comment (plaplant) [inline hera_qm/vis_metrics.py]:
```suggestion
            # remove invalid entries from masked values in true_divide
            corr[np.isclose(corr, 0.0) + (corr < 0) + np.isnan(corr)] = 0.0
```

**Comment by plaplant** [comment]
Great! Thanks for doing the leg work. I didn't know that was the behavior of the `where` argument, and that makes sense why we'd get spurious negative values (because as you've found, `empty` makes no guarantees about the contents of the array, unlike `zeros` or `ones`). I've made a suggestion for making the comment a little more precise. If the new version is okay with you, I'm okay with merging this in.

**Comment by nkern** [comment]
Sure, thanks @plaplant !

**Pull request merged**